main.py

```diff
@@ -59,6 +61,26 @@ def get_city():
 
     return city.capitalize()
 
+def store_data():
+    db_handler = DBHandler('records_db.sqlite')
+
+    db_handler.create_table()# creates table if it does not exist
+
+#the following lines add data using dbhandler methods
+
+db_handler.add_travel_advisory(advisory_data)
+db_handler.addd_food_to_eat(food_data)
+db_handler.add_things_to_do(things_to_do_data)
+db_handler.add_geo_location(geo_location_data)
+
+# finding geo location data
+place_name = "place name"
+geo_location = DataPresenter.get_geolocation_data(place_name)
+print(geo_location)
+
 
 if __name__ == "__main__":
```

Comment:
There are two checks `if __name__ == "__main__"` do you need both?

Reply:
I am only using the last one to store data I am not sure who had added the other one there

Reply:
Which is better?  I forget how exactly this check worked.    If we leave yours for storing data, does that mean the rest of it doesn't work? Actual question, I'm not sure how this works, sorry.